pkg/apis/storage/validation/validation_test.go

```diff
@@ -1696,7 +1696,7 @@ func TestCSIDriverValidation(t *testing.T) {
 	storageCapacity := true
 	notStorageCapacity := false
 	supportedFSGroupPolicy := storage.FileFSGroupPolicy
-	invalidFSGroupPolicy := storage.ReadWriteOnceWithFSTypeFSGroupPolicy
+	invalidFSGroupPolicy := storage.ReadWriteOnceWithFSTypeFSGroupPolicy //nolint:ineffassign
```

Comment:
@msau42 @jsafrane @mattcary Can you or someone storage-ish make a statement on the real fix here?  it's clearly wrong...

Reply:
@huffmanca It looks like you made the change, was this just an alternative to doing `var invalidFSGroupPolicy storage.FSGroupPolicy`?

Reply:
I think we just want `invalidFSGroupPolicy = "invalid-mode"` but maybe there's some type casting issue.

Reply:
found a better way to create a `invalidFSGroupPolicy`

Reply:
Probably `invalidFSGroupPolicy := storage.FSGroupPolicy("invalid-mode")` should work?

Reply:
@mattcary @msau42 i went with the simplest option declare a var of the type. so we should be good. (and removed the typecast)